keras_nlp/backend/config.py

```diff
@@ -12,93 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import json
 import os
 
-_MULTI_BACKEND = False
-_USE_KERAS_3 = False
-
-# Set Keras base dir path given KERAS_HOME env variable, if applicable.
-# Otherwise either ~/.keras or /tmp.
-if "KERAS_HOME" in os.environ:
-    _keras_dir = os.environ.get("KERAS_HOME")
-else:
-    _keras_base_dir = os.path.expanduser("~")
-    if not os.access(_keras_base_dir, os.W_OK):
-        _keras_base_dir = "/tmp"
-    _keras_dir = os.path.join(_keras_base_dir, ".keras")
-
-# Attempt to read KerasNLP config file.
-_config_path = os.path.expanduser(os.path.join(_keras_dir, "keras_nlp.json"))
-if os.path.exists(_config_path):
-    try:
-        with open(_config_path) as f:
-            _config = json.load(f)
-    except ValueError:
-        _config = {}
-    _MULTI_BACKEND = _config.get("multi_backend", _MULTI_BACKEND)
-
-# Save config file, if possible.
-if not os.path.exists(_keras_dir):
-    try:
-        os.makedirs(_keras_dir)
-    except OSError:
-        # Except permission denied and potential race conditions
-        # in multi-threaded environments.
-        pass
-
-if not os.path.exists(_config_path):
-    _config = {
-        "multi_backend": _MULTI_BACKEND,
-    }
-    try:
-        with open(_config_path, "w") as f:
-            f.write(json.dumps(_config, indent=4))
-    except IOError:
-        # Except permission denied.
-        pass
-
-# If KERAS_BACKEND is set in the environment use multi-backend keras.
-if "KERAS_BACKEND" in os.environ and os.environ["KERAS_BACKEND"]:
-    _MULTI_BACKEND = True
-
-
-def detect_if_tensorflow_uses_keras_3():
-    # We follow the version of keras that tensorflow is configured to use.
-    from tensorflow import keras
+from tensorflow import keras
 
-    # Note that only recent versions of keras have a `version()` function.
-    if hasattr(keras, "version") and keras.version().startswith("3."):
-        return True
-
-    # No `keras.version()` means we are on an old version of keras.
-    return False
+# We follow the version of keras that tensorflow is configured to use.
+_USE_KERAS_3 = False
 
+# Note that only recent versions of keras have a `version()` function.
+if hasattr(keras, "version") and keras.version().startswith("3."):
+    _USE_KERAS_3 = True
 
-_USE_KERAS_3 = detect_if_tensorflow_uses_keras_3()
-if _USE_KERAS_3:
-    _MULTI_BACKEND = True
+if not _USE_KERAS_3:
```

Comment:
This is totally optional. We don't need to have it, but since we used to switch to keras core if `KERAS_BACKEND` was set, this seems like a useful signpost.

Reply:
This is good to have IMO!